docs/Bauanleitung/multigeiger-bill-of-material.txt

```diff
@@ -0,0 +1,18 @@
+TEIL	ANZAHL 	BESCHREIBUNG	Artikelbezeichnung	LINK
```

Comment:
fuer deutsche / schweizer leser gut, aber ich denk mal, wir brauchen das gleiche auch auf englisch.

Reply:
hmm, die schweiz ist nicht in der EU - problem bei digikey.ch-bestellungen?

Reply:
stimmt, .com sollte durchgehend möglich sein.

Reply:
dachte zuerst an eine version auf DE und eine auf EN. Bringt aber nicht wirklich etwas. EN sollte reichen.